Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
   "crates/turbo-tasks*",
   "crates/turbopack*",
   "crates/turborepo*",
+  "packages/turbo-repository/rust/",
```

Comment:
Any benefit in including this here? (I'm still a bit fuzzy on what this root Cargo.toml does and what's expected to be in here)

Reply:
Yes, this is the equivalent of adding a workspaces entry but for Rust. Among other things, it allows the `turbo-repository` crate to depend on crates defined under `crates/`